.github/workflows/daily.yaml

```diff
@@ -20,11 +20,6 @@ jobs:
     strategy:
       matrix:
         include:
-        - name: cpu-3.12-2.4
-          container: mosaicml/pytorch:2.4.1_cpu-python3.12-ubuntu22.04
-          markers: not daily and (remote or not remote) and not gpu and not doctest
-          pytest_command: coverage run -m pytest
-          composer_package_name: mosaicml
         - name: cpu-3.12-2.5
```

Comment:
@bowenyang008 @dakinggg, just to clarify, are we okay with removing 2.4 and 2.5? Just wanted to make sure.

Reply:
yes